auto_nag/utils.py

```diff
@@ -622,10 +623,24 @@ def nice_round(val):
 
 
 def get_sort_by_bug_importance_key(bug):
+    """
+    We need bugs with high severity (S1 or S2) or high priority (P1 or P2) to be
+    first (do not need to be high in both). Next, bugs with higher priority and
+    severity are preferred. Finally, for bugs with the same severity and priority,
+    we favour recent bugs. We use the bug ID to reflect the creation order.
```

Comment:
Maybe we could use last_activity time if available, and fallback to ID if last_activity is not available? This way we also consider recently updated bugs and not only recently created. Do as you prefer :)

Reply:
Done in ef5d5910ae87549850c46a9203b9c6823fcbd685